lib/documents/document_store.rb

```diff
@@ -0,0 +1,116 @@
+require 'database/operation_executor'
+require 'documents/conventions'
+require 'requests/request_executor'
+require 'database/exceptions'
+
+module RavenDB
+  class Configuration
+    attr_accessor :default_database, :urls
+  end
+
+  class DocumentStore
+    def initialize(url_or_urls = nil, default_database = nil)
+      @_urls = []  
```

Comment:
@a-serdukov-mobilunity Should we use private `attr_reader` and `attr_writer` for such fields?

Reply:
you wanna just rename @_urls to @ulrs and use attr_reader :urls instead of def urls @_urls end ?

Reply:
I don't know, just asking if that's the right way for getters/setters in Ruby. There's no difference in behavior here, or is there?

Reply:
yes attr_reader :prop just adds on preprocessing

def prop
   @prop
end